tethys_gizmos/static/tethys_gizmos/css/tethys_map_view.css

```diff
@@ -158,6 +158,11 @@
   -ms-transition: height 0.2s ease-in;
   transition: height 0.2s ease-in;
 }
+.tethys-map-view-legend .legend-content .legend-items .legend-item .tethys-mvlegendimage.open {
```

Comment:
tethys_map_view.css is generated from this less file: https://github.com/tethysplatform/tethys/blob/dev/tethys_gizmos/static/tethys_gizmos/less/tethys_map_view.less   Please update the less file and build it, overwriting the css file.